### PR TITLE
[Tests-only] OCIS API tests public link single file

### PR DESCRIPTION
**Pull request by PVince81**
Enabled OCIS API tests related to the public single file share.

For https://github.com/owncloud/ocis-reva/issues/278

Proof for the tests working here https://github.com/owncloud/ocis-reva/pull/314

### CODE REVIEW

**Review by individual-it — APPROVED** [review]

**Review by individual-it — CHANGES_REQUESTED** [review]
the "prove" didn't work https://github.com/owncloud/ocis-reva/pull/314

**Comment by PVince81** [comment]
rebased and adjusted

File: tests/acceptance/features/apiSharePublicLink1/accessToPublicLinkShare.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@api @TestAlsoOnExternalUserBackend @public_link_share-feature-required @files_sharing-app-required @skipOnOcis @issue-ocis-reva-49 @issue-ocis-reva-282
+@api @TestAlsoOnExternalUserBackend @public_link_share-feature-required @files_sharing-app-required @issue-ocis-reva-49 @issue-ocis-reva-282
```
Comment (PVince81) [inline tests/acceptance/features/apiSharePublicLink1/accessToPublicLinkShare.feature:1]: oh, I removed too much...

**Comment by PVince81** [comment]
the proof is green: https://github.com/owncloud/ocis-reva/pull/314

**Review by individual-it — APPROVED** [review]

**Pull request merged**